odoo/tests/result.py

```diff
@@ -80,6 +80,8 @@ def __init__(self, stream=None, descriptions=None, verbosity=None):
         self._soft_fail = False
         self.had_failure = False
         self.stats = collections.defaultdict(Stat)
+        self.global_report = global_report
+        self.fail_fast = False
```

Comment:
what is this attribute?

Reply:
It allows to log one error when we fail fast and not one per test suite